webpack.config.js

```diff
@@ -10,6 +10,9 @@ module.exports = {
     filename: '[name].js',
   },
   plugins: [
+    new webpack.DefinePlugin({
+      __DEV__: JSON.stringify(process.env.NODE_ENV !== "production"),
+    }),
```

Comment:
Apollo Client has moved away from using `process.env.NODE_ENV` to `__DEV__` (details [here](https://github.com/apollographql/apollo-client/pull/8347)). This slight tweak is intended to help Webpack recognize that.